napari/_qt/qt_main_window.py

```diff
@@ -474,23 +475,20 @@ def _help_changed(self, event):
         """
         self._help.setText(event.text)
 
-    def _screenshot_dialog(self):
+    def _screenshot_dialog(self, *, default_name=""):
```

Comment:
I think you can drop the `default_name` now

Reply:
@Czaki thoughts?

Reply:
removed